service/matching/matchingEngine.go

```diff
@@ -275,9 +278,15 @@ pollLoop:
 				return emptyPollForDecisionTaskResponse, nil
 			}
 
+			isStickyEnabled := false
+			if describeResp.ExecutionConfiguration.StickyTaskList != nil && len(describeResp.ExecutionConfiguration.StickyTaskList.GetName()) != 0 {
```

Comment:
I see this check in multiple places.  Can we just have an API on ExecutionConfiguration for StickyTasklist?